src/libraries/System.Text.Json/src/System/Text/Json/Writer/JsonWriterHelper.cs

```diff
@@ -36,6 +36,13 @@ public static void WriteIndentation(Span<byte> buffer, int indent, byte indentBy
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static void ValidateNewLine(string value)
+        {
+            if (value is not JsonConstants.NewLineLineFeed and not JsonConstants.NewLineCarriageReturnLineFeed)
```

Comment:
If someone wants to explicitly set the default behavior, how do they do that? Shouldn't null be allowed as well to indicate default?

Reply:
Setting to `Environment.NewLine` would work. I don't have a strong opinion, but there is precedent in returning the default setting in the other properties of JSO.